### DISCUSSION TITLE
Translation Support for Non-English Users

### DISCUSSION THREAD

**Discussion by goleedev** [facebook/stylex#191] [General]
Are there any plans to provide translated documents for users who do not speak English? If not, I would like to initiate the translation process myself, starting with Korean, which is my first language!

**Comment by timwehrle** [comment]
I agree with this. Would also contribute to other languages.

**Comment by nmn** [comment]
Yes! We would love contributions to translate the documentation! I'll try to update the project setup to make this process easier.

**Reply by goleedev** [reply]
Yay! Thanks in advance 🥳

**Comment by nmn** [comment]
I found a Korean translation of our launch blog post:

https://emewjin.github.io/introducing-stylex/

----

I'm still working on improving the website to allow translations on the official website. But would love contributions as a collection of links here in the meantime.

**Reply by goleedev** [reply]
Absolutely! I'll start gathering related links right here.
